docs/source/modules/benchmark_tutorial.rst

```diff
@@ -44,13 +44,16 @@ If your code depends on additional requirements, it is good practice to addition
 or :code:`setup.py` file specifying the dependencies.
 
 Before executing the packaging methods to actually upload to S3, please check in with us
-(msch@mit.edu, mferg@mit.edu, jjpr@mit.edu) so that we can give you access. With the credentials, you can then
+(msch@mit.edu, mferg@mit.edu) so that we can give you access. With the credentials, you can then
```

Comment:
```suggestion
(via [Slack or a Github issue](https://www.brain-score.org/community)) so that we can give you access. With the credentials, you can then
```